core/src/main/java/org/apache/carbondata/core/datastore/filesystem/LocalCarbonFile.java

```diff
@@ -178,6 +178,25 @@ public boolean delete() {
     return carbonFiles;
   }
 
+  @Override public List<CarbonFile> listFiles(Boolean recursive, CarbonFileFilter fileFilter)
```

Comment:
recursive variable is not used in the logic

Reply:
fixed